pvr.vuplus/changelog.txt

```diff
@@ -1,3 +1,12 @@
+v2.4.7 
```

Comment:
`addon.xml.in` says the version is `2.4.9`, do you think you could change it here too so they match?